src/main/java/net/fabricmc/loom/configuration/providers/mappings/IntermediaryMappingsProvider.java

```diff
@@ -107,6 +110,24 @@ public void execute(final DependencyArtifact dependencyArtifact) {
 
 	@Override
 	public @NotNull String getName() {
+		final String encodedMcVersion = UrlEscapers.urlFragmentEscaper().escape(getMinecraftVersion().get());
+		final String urlRaw = getIntermediaryUrl().get();
+		if (!LoomGradleExtensionApiImpl.DEFAULT_INTERMEDIARY_URL.equals(urlRaw)) {
```

Comment:
This wont build due to checkstyle, please make sure it builds and actually works by testing it.

I dont see an automated test for this (likely becuase its quite hard to test), so it definitely should be manually tested.